pkg/api/vlabs/validate_test.go

```diff
@@ -253,6 +253,20 @@ func Test_OrchestratorProfile_Validate(t *testing.T) {
 				},
 			},
 		},
+		"should error when enableEncryptionWithExternalKms=true with own key and all required parameters not provided": {
```

Comment:
I know this is annoying, but we should probably UT the remaining 4 non-valid permutations. (did I do the math right???)